### DISCUSSION TITLE
How to update Node.js to the latest Current release

### DISCUSSION THREAD

**Discussion by chalkygames123** [renovatebot/renovate#11180] [Request Help] [answered]
How do I receive a PR to update Node.js to the latest Current release (currently 16.6.1)?

The test repository is https://github.com/chalkygames123/test-renovate-node, and I have specified the pinned version `16.0.0` in the `engines.node` and `volta.node` fields in `package.json` and also in `.nvmrc`.

Adding the `supportPolicy` option to the configuration is not helpful. Am I misunderstanding how that option works?

**Comment by viceice** [comment, marked as answer]
Yes, support policy only works for Travis and will be removed in v26


https://github.com/renovatebot/renovate/pull/11176

We use our custom node versioning, which only accepts lts versions as stable. So to get non lts updates you need to disable `ignoreUnstable` option for node with a package rule.

**Reply by rarkins** [reply]
Yes, please set `ignoreUnstable=true`. Today if we detect the current version is unstable then we only upgraded to a newer unstable version automatically if it has the same major.minor.patch (e.g. 2.0.0-alpha.7 -> 2.0.0.beta.1)

**Reply by viceice** [reply]
It's a little strange,default value for `ignoreUnstable` is undefined, but `:ignoreUnstable` preset toggles this via `config:base` preset to `true`.

Do not set `ignoreUnstable=false` globally, otherwise renovate will update all repo deps to latest unstable versions.


So the following should allow to update to latrest node version
```json
{
  "packageRules": [
    {
       "matchpackageNames": ["node"],
       "ignoreUnstable": false
    }
  ]
}
```

**Reply by viceice** [reply]
ref #11185

**Reply by rarkins** [reply]
boolean default to `true` actually. It's a weird historical thing we should change.

**Reply by chalkygames123** [reply]
Okay, thank you all. The config that @viceice suggested above does the trick! I'll keep my eye on linked refs 👀
